src/styles/bloks/cards.scss

```diff
@@ -0,0 +1,124 @@
+.cards {
+  --cards-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--cards-count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    & {
+      --cards-count: 2;
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      --cards-count: 3;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      --cards-count: 4;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  color: $color-nav;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
It's better to define the font-family in a more general scope, like in the body tag. This way, you ensure that if any other element needs to use the same font, it will inherit it from the body. This is a recommendation, not a requirement.